src/AttributesAndMethods.jl

```diff
@@ -537,60 +569,60 @@ julia> projective_space( 2 ) * projective_space( 2 )
 NormalToricVariety(GAP: <A projective toric variety of dimension 4 which is a product of 2 toric varieties>, Polymake.BigObjectAllocated(Ptr{Nothing} @0x00005645a1a00930))
 ```
 """
-function Base.:*( v::normalToricVariety, w::normalToricVariety )
+function Base.:*( v::NormalToricVariety, w::NormalToricVariety )
     gap_NormalToricVariety = v.GapNTV * w.GapNTV
     return NormalToricVariety( gap_NormalToricVariety )
 end
 export *
 
 
-struct characterToRationalFunction
+struct CharacterToRationalFunction
            gap_CharacterToRationalFunction::GapObj
 end
-export characterToRationalFunction
+export CharacterToRationalFunction
 
 
 """
-    character_to_rational_function( l::Vector{Int}, v::normalToricVariety )
+    character_to_rational_function( l::Vector{Int}, v::NormalToricVariety )
 
 Turn the character `l` of the normal toric variety `v` into a rational function.
 """
-function character_to_rational_function( l::Vector{Int}, v::normalToricVariety )
+function character_to_rational_function( l::Vector{Int}, v::NormalToricVariety )
     gap_CharacterToRationalFunction = GAP.Globals.CharacterToRationalFunction( GapObj( l ), v.GapNTV )
-    return characterToRationalFunction( gap_CharacterToRationalFunction )
+    return CharacterToRationalFunction( gap_CharacterToRationalFunction )
 end
 export character_to_rational_function
 
 
 """
-    blowup_on_ith_minimal_torus_orbit( v::normalToricVariety, i::Int )
+    blowup_on_ith_minimal_torus_orbit( v::NormalToricVariety, i::Int )
 
 Compute the blow up of the normal toric variety `v` on the i-th minimal torus orbit.
 """
-function blowup_on_ith_minimal_torus_orbit( v::normalToricVariety, i::Int )
+function blowup_on_ith_minimal_torus_orbit( v::NormalToricVariety, i::Int )
     gap_blowup_variety = GAP.Globals.BlowUpOnIthMinimalTorusOrbit( v.GapNTV, GapObj( i ) )
     return NormalToricVariety( gap_blowup_variety )
 end
 export blowup_on_ith_minimal_torus_orbit
 
 
 """
-    ith_betti_number( v::normalToricVariety, i::Int )
+    ith_betti_number( v::NormalToricVariety, i::Int )
 
 Compute the i-th Betti number of the normal toric variety `v`.
 """
-function ith_betti_number( v::normalToricVariety, i::Int )
+function ith_betti_number( v::NormalToricVariety, i::Int )
     return GAP.Globals.ithBettiNumber( v.GapNTV, GapObj( i ) )::Int
```

Comment:
Note that the GapObj is not necessary here, an Int can always be passed to a GAP function